src/MigrationTools/Services/TelemetryClientAdapter.cs

```diff
@@ -57,6 +68,27 @@ public void TrackEvent(string name, IDictionary<string, string> properties, IDic
         public void TrackException(Exception ex, IDictionary<string, string> properties, IDictionary<string, double> measurements)
```

Comment:
What's in `measurements`? Maybe it would make sense to add those to the elmah.io message?

Reply:
Ooohhh ... god idea... 

There is properies and measurement...

Reply:
![image](https://github.com/nkdAgility/azure-devops-migration-tools/assets/5205575/ca7227cf-a7dd-47b0-b4f2-98e88655fce2)
![image](https://github.com/nkdAgility/azure-devops-migration-tools/assets/5205575/238c64b6-37a1-4b1f-9a66-db66d766ac93)

Reply:
I added both to the Data collection.

Reply:
I would add them to `Data`. Not sure what `CollectionUrl` is. But there's a property on `CreateMessage` called `Url`. Maybe use that? If not, use `Data` 😄